CHANGELOG.md

```diff
@@ -61,12 +61,21 @@
 * Added a `Realm.GetAllSubscriptions()` extension method that allows you to obtain a collection of all registered query-based sync subscriptions. ([Issue #1838](https://github.com/realm/realm-dotnet/issues/1838))
 * Added `AccessLevel` property to `PathPermission` to replace the now deprecated `MayRead/MayWrite/MayManage`. ([Issue #1863](https://github.com/realm/realm-dotnet/issues/1863))
 * Added `RealmOwnerId` property to `PathPermission` that indicates who the owner of the Realm is. ([Issue #1863](https://github.com/realm/realm-dotnet/issues/1863))
+* Added support for building with `dotnet build` (previously only the `msbuild` command line was supported). ([PR #1849](https://github.com/realm/realm-dotnet/pull/1849))
+* Improved query performance for unindexed string columns when the query has a long chain of OR conditions. (Core upgrade)
+* Improved performance of encryption and decryption significantly by utilizing hardware optimized encryption functions. (Core upgrade)
+* Compacting a realm into an encrypted file could take a really long time. The process is now optimized by adjusting the write buffer size relative to the used space in the realm. (Core upgrade)
+* The string-based query parser (`results.Filter("...")`) now supports readable timestamps with a 'T' separator in addition to the originally supported "@" separator. For example: `"startDate > 1981-11-01T23:59:59:1"`. (Core upgrade)
```

Comment:
Forgive my ignorance - does .Net also support string based queries now?

Reply:
Yes - we added support in 3.0.0 as we didn't want to hold functionality hostage to our LINQ support: https://github.com/realm/realm-dotnet/blob/master/CHANGELOG.md#enhancements-5

Reply:
Cool - Where does one find docs for that? (I did actually try to find it without success)